pgmpy/base/UndirectedGraph.py

```diff
@@ -174,3 +174,29 @@ def is_triangulated(self):
         True
         """
         return nx.is_chordal(self)
+
+    def fill_in_edges(self, variables):
+        """
+        Return edges needed to be added to the graph if a variable is remvoed.
+
+        Parameters
+        ----------
+        variables: list of nodes, or one node name.
+            Node or nodes to be removed from the graph.
+
+        Examples
+        --------
+        >>> from pgmpy.readwrite.BIF import BIFReader
+        >>> reader = BIFReader("datasets/asia.bif")
+        >>> bn = reader.get_bayesian_model()
+        >>> ug = bn.moralize()
+        >>> print(ug.fill_in_edges("bronc"))
+        [('dysp', 'smoke'), ('either', 'smoke')]
+        """
+        if not isinstance(variables, (list, set)):
+            variables = [variables]
+        edges = []
+        for variable in variables:
+            neighbors = self.neighbors(variable)
+            edges = [e for e in itertools.combinations(neighbors, 2) if not self.has_edge(*e)]
+        return edges
```

Comment:
@jhonatanoliveira Correct me if I am wrong. The method ```fill_in_edges``` takes input a ```list``` of nodes to be removed. But by looking into the code it seems that the method would return only the edges to be removed only for the last variable as ```edges``` variable is updated in each iteration without storing the previous results.

Furthermore could you please write the tests for each of the method that you wrote with all possible test cases. That would make these methods more robust.